static/css/taxbrain.css

```diff
@@ -8440,6 +8440,18 @@ label {
   border-right: 2px solid #dddddd;
   padding-left: 30px;
 }
+.table-check-all {
```

Comment:
yes, this was new to me as well. See the conversation with @mistakevin on Tax-Calculator Flowdock